ui/src/app/views/signed-in/component.html

```diff
@@ -51,8 +51,8 @@ <h6 style="padding: 0.5rem;">
           [class.active]="homeActive"
           routerLink="/"
           (click)="sidenavToggle = false">Home</button>
-      <button class="btn btn-primary nav-link"
-          (click)="openDataBrowser()">Browse Data</button>
+      <a class="btn btn-primary nav-link"
```

Comment:
I like this change; if you haven't already, I'd double check the styling wasn't affected though

Reply:
I did double check, looks good.